lib/tasks/say.js

```diff
@@ -239,17 +242,14 @@ class TaskSay extends TtsTask {
       label = fallbackLabel;
     }
 
-    let filepath;
-    try {
-      filepath = await this._synthesizeWithSpecificVendor(cs, ep, {vendor, language, voice, label});
-    } catch (error) {
+    const startFallback = async(error) => {
       if (fallbackVendor && this.isHandledByPrimaryProvider && !cs.hasFallbackTts) {
         this.notifyError(
           { msg: 'TTS error', details:`TTS vendor ${vendor} error: ${error}`, failover: 'in progress'});
         this.isHandledByPrimaryProvider = false;
         cs.hasFallbackTts = true;
         this.logger.info(`Synthesize error, fallback to ${fallbackVendor}`);
-        filepath = await this._synthesizeWithSpecificVendor(cs, ep,
+        this._filepath = await this._synthesizeWithSpecificVendor(cs, ep,
```

Comment:
why is it necessary to make this an instance variable now?

Reply:
You are right, I was bad here, revert it to local variable